index.ts

```diff
@@ -12,6 +11,47 @@ const autocompleteUrl = `${mapzenUrl}/autocomplete`
 const reverseUrl = `${mapzenUrl}/reverse`
 const searchUrl = `${mapzenUrl}/search`
 
+type Rect = {
+  maxLat: number
+  maxLon: number
+  minLat: number
+  minLon: number
+}
+type Circle = {
+  centerPoint: number
+  radius: number
+}
+type Boundary = {
+  circle?: Circle
+  country: string
+  rect: Rect
+}
+
+type Query = {
+  // apiKey is renamed depending on if it is being passed around or into fetch
+  // OK to disable this rule as api_key is a hack to conform with api
+  // eslint-disable-next-line camelcase
+  api_key: string
+  apiKey?: string
+  boundary?: Boundary
+  focusPoint?: LonLatInput
+  format?: boolean
+  layers?: string
+  options?: RequestInit // Built-in Typing
+  point?: GeoJSON.Point | LonLatOutput
+  size?: number
+  sources?: string
+  text?: string
+  url?: string
+}
+
+type MapzenInput = {
```

Comment:
This name feels weird to me. Maybe `PeliasFetchArgs` instead? (Mapzen no longer exists, even though it's still the name of the repo.)

Reply:
Good call! Renamed in 165129c2154d6b79a50f8b25ee3295a757fccfee